src/Illuminate/Database/DatabaseManager.php

```diff
@@ -280,7 +280,12 @@ public function supportedDrivers()
      */
     public function availableDrivers()
     {
-        return array_intersect($this->supportedDrivers(), PDO::getAvailableDrivers());
+        return array_unique(
```

Comment:
why do we need array_unique - surely the intersection of the two has no duplicate elements?